src/app/components/homeComponents/currencySelector/currencySelectorComponents/SliderArrows.tsx

```diff
@@ -0,0 +1,21 @@
+import { NextPrevBtnSVG } from "../../../../../../public/currencySelectorSVGS";
+
+export const PrevArrow = ({ onClick }: { onClick: () => void }) => (
+  <button
+    className="z-20 rotate-180 absolute transform -translate-y-1/2"
+    onClick={onClick}
+    style={{ top: "45%", right: "97%" }}
```

Comment:
why inline styling?

Reply:
Trying to position them within tailwind wasn't working, noticed that it worked with inline styling